components/atoms/Highlighter/index.tsx

```diff
@@ -1,56 +1,58 @@
-import CopyToClipboard from '@components/atoms/CopyToClipboard'
-import PrismLibrary from '@languages/index'
 import createRandomId from '@util/createRandomId'
-import returnHighlightIndices from '@util/returnHighlightIndices'
-import toStringSet from '@util/toStringSet'
-import Highlight, { defaultProps, Language } from 'prism-react-renderer'
-import theme from 'prism-react-renderer/themes/nightOwlLight'
-import { FunctionComponent, createRef } from 'react'
+import { FunctionComponent } from 'react'
 
-import { 
-    StyledHighlighterMatch,
+import {
     StyledCodeBlock,
+    StyledHighlighterMatch,
     StyledCodeWrapper,
-    StyledHighlighterWrapper 
+    StyledNonHighlighterMatch,
 } from './HighlighterStyles'
 
 interface Props {
     input: string
-    language: Language
+    prismSyntax?: boolean
```

Comment:
Basically, a way for us to preserve the `color` settings from #372 without also overriding styles from Prism.